ko/admin/config.rst

```diff
@@ -391,6 +391,8 @@ CUBRID는 데이터베이스 서버, 브로커, CUBRID 매니저로 구성된다
 |                               | deduplicate_key_level               | 클라이언트/서버         |         | int      | -1                             |                 |
 |                               +-------------------------------------+-------------------------+---------+----------+--------------------------------+-----------------+
 |                               | print_index_detail                  | 클라이언트/서버         |         | bool     | no                             |                 |
+|                               +-------------------------------------+-------------------------+---------+----------+--------------------------------+-----------------+
+|                               | enable_memory_monitoring            | 클라이언트/서버         |         | bool     | no                             |                 |
```

Comment:
Isn't this just a server parameter?

Reply:
@joohok 
Other utilities that are affected by the server's system parameters are currently expressed in the same way.

Reply:
@Rudeus 
In my opinion, this parameter should be specified as "서버" and not "클라이언트/서버", since it only reflects on the server.

Reply:
@Rudeus 
Is this parameter also used in the memmon utility (client) ?

Reply:
For background :
As far as I know, the execution of the `memmon` utility is determined by the value of this system parameter.
( e.g. if `enable_memory_monitoring` is set to false, `memmon` utility will not be  executed) 

The utility is a client, and since the client's operation is affected by this system parameter, I had suggested specifying it as a client/server system parameter.